packages/base/src/commands.ts

```diff
@@ -83,7 +83,9 @@ export function setVisible(
   if (object) {
     object.visible = value;
```

Comment:
Why the `visible` value here does not work?

Reply:
It works and as far as I can notice, it updates the visible property in the `.jcad` document, however it doesn't instantly updates the state of the model hence I have to also use sharedModel to directly update the state on the fly.

Reply:
something like this is enough, `updateObjectByName` does check for object existence already

```ts
export function setVisible(
  sharedModel: IJupyterCadDoc,
  name: string,
  value: boolean
) {
  sharedModel.updateObjectByName(name, {
    data: { key: 'visible', value }
  });
}
```

Reply:
Done in [`da3bac8`](https://github.com/jupytercad/JupyterCAD/pull/413/commits/da3bac8f6d773cb37cf0ec3ff9906fd172a71b85)